System.Security.Cryptography/AES.cs

```diff
@@ -0,0 +1,162 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System;
+
+namespace System.Security.Cryptography
+{
+    /// <summary>
+    /// Represents the class for aes encryption/decryption
+    /// </summary>
+    public class AES
+    {
+        public enum EncryptionModes { ECB }
```

Comment:
Is this aligned with the Full .NET ?
And is this enum also part of the class or outside of the class?
In all cases, you have to add intellisesne comments on it

Reply:
I made some changes here.
the Full .Net uses this "Represents the abstract base class from which all implementations of the Advanced Encryption Standard (AES) must inherit." but this implementation doesn't use abstract class.
should we set excactly Full .Net comments here?

Reply:
>should we set excactly Full .Net comments here?

No, it's ok :-) Important is mainly the definitions of the functions and the intellisense comments associated to them.

What about the enum?